cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,162 @@
+/// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('user clicks on Image content type', () => {
+    cy.get(`ul.admin-list span.label:contains('Image')`).then($el => {
+        const content= $el[1];
```

Comment:
this step exists in common - Image type should not be hardcoded

Reply:
done